src/MigrationTools.Host/MigrationToolHost.cs

```diff
@@ -33,6 +33,16 @@ public static IHostBuilder CreateDefaultBuilder(string[] args)
                 return null;
             }
 
+            bool SendTelemitery = true;
+            if (executeOptions is not null)
+            {
+                if (executeOptions.Telemetry is not null)
```

Comment:
ifs can be collapsed to one if with executeOptions?.Telemetry is not null

Reply:
If it is not passed then it is null